actions/banked-months-updater.vbs

```diff
@@ -0,0 +1,482 @@
+                                            "The script has stopped. Please check your Internet connection. Consult a scripts administrator with any questions.", _
+                                            vbOKonly + vbCritical, "BlueZone Scripts Critical Error")
+            StopScript
+		END IF
+	ELSE
+		FuncLib_URL = "C:\MAXIS-scripts\MASTER FUNCTIONS LIBRARY.vbs"
+		Set run_another_script_fso = CreateObject("Scripting.FileSystemObject")
+		Set fso_command = run_another_script_fso.OpenTextFile(FuncLib_URL)
+		text_from_the_other_script = fso_command.ReadAll
+		fso_command.Close
+		Execute text_from_the_other_script
+	END IF
+END IF
+'END FUNCTIONS LIBRARY BLOCK================================================================================================
+
+'CHANGELOG BLOCK ===========================================================================================================
+'Starts by defining a changelog array
+changelog = array()
+
+'INSERT ACTUAL CHANGES HERE, WITH PARAMETERS DATE, DESCRIPTION, AND SCRIPTWRITER. **ENSURE THE MOST RECENT CHANGE GOES ON TOP!!**
+'Example: call changelog_update("01/01/2000", "The script has been updated to fix a typo on the initial dialog.", "Jane Public, Oak County")
+call changelog_update("09/13/2023", "Initial version.", "Ilse Ferris, Hennepin County")
+
+'Actually displays the changelog. This function uses a text file located in the My Documents folder. It stores the name of the script file and a description of the most recent viewed change.
+changelog_display
+'END CHANGELOG BLOCK =======================================================================================================
+
+'THE SCRIPT----------------------------------------------------------------------------------------------------
+EMConnect ""
+Call check_for_MAXIS(False)
+Call MAXIS_case_number_finder(MAXIS_case_number)
+Call MAXIS_footer_finder(initial_month, initial_year)
+
+Dialog1 = ""
+BeginDialog Dialog1, 0, 0, 186, 90, "Case Number/Date Selection Dialog"
+  EditBox 75, 5, 45, 15, MAXIS_case_number
+  EditBox 75, 25, 20, 15, initial_month
+  EditBox 100, 25, 20, 15, initial_year
+  EditBox 75, 45, 95, 15, worker_signature
+  ButtonGroup ButtonPressed
+    OkButton 65, 65, 50, 15
+    CancelButton 120, 65, 50, 15
+  Text 10, 30, 65, 10, "Initial month/year:"
+  Text 25, 10, 50, 10, "Case Number: "
+  Text 10, 50, 60, 10, "Worker Signature:"
+EndDialog
+
+DO
+	DO
+		err_msg = ""
+		dialog Dialog1
+		cancel_without_confirmation
+		Call validate_MAXIS_case_number(err_msg, "*")
+		Call validate_footer_month_entry(initial_month, initial_year, err_msg, "*")
+		If trim(worker_signature) = "" then err_msg = err_msg & vbNewLine & "* Sign your case note."
+		IF err_msg <> "" THEN MsgBox "*** NOTICE!!! ***" & vbCr & err_msg & vbCr & vbCr & "Please resolve for the script to continue."
+	LOOP UNTIL err_msg = ""
+call check_for_password(are_we_passworded_out)
+Loop until are_we_passworded_out = false
+
+Call_check_for_MAXIS(False)
```

Comment:
Extra underscore after Call causes script to error out.

Reply:
Oh my gosh, I've done this SO much on this script! Never have I ever had this many problems with it, ha!